nailgun/entities.py

```diff
@@ -3971,6 +4195,43 @@ def update_payload(self, fields=None):
         }
 
 
+class SmartClassParameters(
+        Entity,
+        EntityReadMixin,
+        EntitySearchMixin,
+        EntityUpdateMixin):
+    """A representation of a Smart Class Parameters."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'puppetclass': entity_fields.OneToOneField(PuppetClass),
+            'override': entity_fields.BooleanField(),
+            'description': entity_fields.StringField(),
+            'default_value': entity_fields.StringField(),
+            'hidden_value': entity_fields.BooleanField(),
```

Comment:
Then, hammer doesn't work in the same way:
https://bugzilla.redhat.com/show_bug.cgi?id=1371794.
In that case, we have really different parameters:
`hidden-value?` is a flag and `hidden-value` is a really value which is not correct behavior as it should substitute default-value with `******`